### PR TITLE
Changes from drive practice on 3/23/2023

### PR DESCRIPTION
**Pull request by jamesdooley4**
Currently targeted at visionimprovements branch.  Will retarget once that branch is merged to main

### CODE REVIEW

**Comment by jamesdooley4** [comment]
Blocking until it's retargeted at the main branch

File: src/main/java/frc/team2412/robot/Controls.java
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/Controls.java:1]: Leaving a comment to remember to retarget this PR

File: src/main/java/frc/team2412/robot/Controls.java [hunk 1]
```diff
@@ -123,12 +123,13 @@ public void bindArmControls() {
 				new ManualArmOverrideOnCommand(
 						s.armSubsystem, codriveController::getRightY, codriveController::getLeftY));
 		armManualControlOff.onTrue(new ManualArmOverrideOffCommand(s.armSubsystem));
-		armLowButton.onTrue(new SetFullArmCommand(s.armSubsystem, ARM_LOW_POSITION, WRIST_RETRACT));
+		armLowButton.onTrue(
+				new SetFullArmCommand(s.armSubsystem, ARM_LOW_POSITION, WRIST_RETRACT, 0.5));
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/Controls.java:127]: The `0.5` here and below ought to be a named constant so it's clear to code readers what it's for

File: src/main/java/frc/team2412/robot/commands/intake/IntakeSetSlowOutCommand.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+	@Override
+	public void initialize() {
+		intakeSubsystem.intakeSlowOut();
+	}
+
+	// TODO: uncomment when color matching works
+	@Override
+	public boolean isFinished() {
+		// return !intakeSubsystem.hasObject();
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/commands/intake/IntakeSetSlowOutCommand.java:22]: I don't think color matching is ever coming back, so recommend deleting the comment & commented-out code

File: src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java [hunk 3]
```diff
@@ -86,10 +86,10 @@ public static class ArmConstants {
...
 		// Trapezoid Profile
 		public static final double MAX_ARM_VELOCITY = 1;
-		public static final double MAX_ARM_ACCELERATION =
-				ARM_FORWARD_LIMIT
-						/ ARM_MOTOR_TO_SHOULDER_ENCODER_RATIO
-						/ ARM_ROTATIONS_TO_SHOULDER_ENCODER_RATIO; // arm rotations / second^2
+		public static final double MAX_ARM_ACCELERATION = 0.5;
+		// ARM_FORWARD_LIMIT
+		// 		/ ARM_MOTOR_TO_SHOULDER_ENCODER_RATIO
+		// 		/ ARM_ROTATIONS_TO_SHOULDER_ENCODER_RATIO; // arm rotations / second^2
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java:92]: Should delete commented-out code or add an explanation as a comment why it's valuable to keep as reference

**Comment by jbko6** [comment]
this will resolve #69

**Review by TAKBS2412 — APPROVED** [review]

**Pull request merged**